app.py

```diff
@@ -30,15 +30,52 @@
 from plugins.messages import MessageDatabase
 from plugins.workshop import Workshop
 
+
 class PSBot(PokemonShowdownBot):
+    """Mainly a wrapper class for the Robot class, implementing required methods.
+
+    This implements the major method required, splitMessage class, and also handles
+    the delegation of tasks to their respective modules. Most of the underlying
+    functionality and/or function calls can be found in the PokemonShowdownBot
+    class in robot.py.
+
+    You should expect to handle the PS protocols outlined here:
+    https://github.com/Zarel/Pokemon-Showdown/blob/master/PROTOCOL.md
+
+    For the sake of simplicity, we assume that the command char used is ~ in the documentation.
+
+    Attributes:
```

Comment:
Most of that documentation is barely relevant anymore, it's a leftover from when the user object wasn't a class of its own. Proper documentation of the user object should be in the user.py file, not here.